### PR TITLE
Initialize Visibility & Storage Gaps

### PR DESCRIPTION
**Pull request by novaknole**
## Description

Please include a summary of the change and be sure you follow the contributions rules we do provide [here](./CONTRIBUTIONS.md)

Task: [ID]()

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] I have selected the correct base branch.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] Any dependent changes have been merged and published in downstream modules.
- [ ] I ran all tests with success and extended them if necessary.
- [ ] I have updated the ``CHANGELOG.md`` file in the root folder.
- [ ] I have tested my code on the test network.

### CODE REVIEW

File: packages/contracts/src/plugins/governance/admin/Admin.sol [hunk 1]
```diff
@@ -71,4 +71,9 @@ contract Admin is IMembershipContract, PluginCloneable, Proposal {
         });
         _executeProposal(dao, proposalId, _actions, _allowFailureMap);
     }
+
+    /// @dev This empty reserved space is put in place to allow future versions to add new
+    /// variables without shifting down storage in the inheritance chain.
+    /// https://docs.openzeppelin.com/contracts/4.x/upgradeable#storage_gaps
```
Comment (Michael-A-Heuer) [inline packages/contracts/src/plugins/governance/admin/Admin.sol:77]:
```suggestion
    /// @notice This empty reserved space is put in place to allow future versions to add new variables without shifting down storage in the inheritance chain (see [OpenZepplins guide about storage gaps](https://docs.openzeppelin.com/contracts/4.x/upgradeable#storage_gaps)).
```
This should be a `/// @notice` to be formatted correctly in our dev-portal. Please make it a one-line comment.

File: packages/contracts/src/token/ERC20/governance/GovernanceERC20.sol [hunk 2]
```diff
@@ -99,4 +99,9 @@ contract GovernanceERC20 is
             _delegate(to, to);
         }
     }
+
+    /// @dev This empty reserved space is put in place to allow future versions to add new
+    /// variables without shifting down storage in the inheritance chain.
+    /// https://docs.openzeppelin.com/contracts/4.x/upgradeable#storage_gaps
```
Comment (Michael-A-Heuer) [inline packages/contracts/src/token/ERC20/governance/GovernanceERC20.sol:105]:
```suggestion
   /// @notice This empty reserved space is put in place to allow future versions to add new variables without shifting down storage in the inheritance chain (see [OpenZepplins guide about storage gaps](https://docs.openzeppelin.com/contracts/4.x/upgradeable#storage_gaps)).
```

File: packages/contracts/src/token/ERC20/governance/GovernanceWrappedERC20.sol [hunk 3]
```diff
@@ -121,4 +121,9 @@ contract GovernanceWrappedERC20 is
     ) internal override(ERC20VotesUpgradeable, ERC20Upgradeable) {
         super._burn(account, amount);
     }
+
+    /// @dev This empty reserved space is put in place to allow future versions to add new
+    /// variables without shifting down storage in the inheritance chain.
+    /// https://docs.openzeppelin.com/contracts/4.x/upgradeable#storage_gaps
```
Comment (Michael-A-Heuer) [inline packages/contracts/src/token/ERC20/governance/GovernanceWrappedERC20.sol:127]:
```suggestion
    /// @notice This empty reserved space is put in place to allow future versions to add new variables without shifting down storage in the inheritance chain (see [OpenZepplins guide about storage gaps](https://docs.openzeppelin.com/contracts/4.x/upgradeable#storage_gaps)).
```

**Review by Michael-A-Heuer — COMMENTED** [review]
Maybe rename the PR? It is not about "add plugin address"

**Review by Michael-A-Heuer — APPROVED** [review]

**Review by brickpop — APPROVED** [review]

**Pull request merged**